### PR TITLE
CTCP-6008: PR to remove href and use formHelper submit

### PR DESCRIPTION
**Pull request by justega247**

### CODE REVIEW

File: test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala [hunk 1]
```diff
@@ -121,9 +121,33 @@ class IntentionToControlP5ControllerSpec extends SpecBase with AppWithDefaultMoc
...
+            message =>
+              val intentionToControlInformationRequestedController: String =
+                controllers.departureP5.routes.IntentionToControlP5Controller.informationRequested(departureIdP5, messageId).url
+
+              when(mockDepartureP5MessageService.getMessage[CC060CType](any(), any())(any(), any(), any())).thenReturn(Future.successful(message))
+              when(mockDepartureP5MessageService.getDepartureReferenceNumbers(any())(any(), any()))
+                .thenReturn(Future.successful(DepartureReferenceNumbers(lrn.value, None)))
+              when(mockReferenceDataService.getCustomsOffice(any())(any(), any())).thenReturn(Future.successful(Right(customsOffice)))
+              when(mockIntentionToControlP5ViewModelProvider.apply(any())(any()))
+                .thenReturn(IntentionToControlP5ViewModel(sections, requestedDocuments = true, Some(lrn.toString)))
```
Comment (jackheslop96) [inline test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala:141]: Are these mocks used?

File: test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala [hunk 2]
```diff
@@ -121,9 +121,33 @@ class IntentionToControlP5ControllerSpec extends SpecBase with AppWithDefaultMoc
...
           contentAsString(result) mustEqual
-            view(intentionToControlP5ViewModel, departureIdP5, customsOfficeContactViewModel)(request, messages).toString
+            view(intentionToControlP5ViewModel, departureIdP5, messageId, customsOfficeContactViewModel)(request, messages).toString
       }
     }
-  }
 
+    "must redirect to Presentation notification frontend" in {
+      forAll(listWithMaxLength[RequestedDocumentType]()) {
+        requestedDocuments =>
```
Comment (jackheslop96) [inline test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala:130]: Very minor but the just the arbitrary CC060Type should suffice here as there's no conditional logic on the requested documents in the onSubmit

File: app/views/departureP5/IntentionToControlP5View.scala.html [hunk 3]
```diff
@@ -31,37 +30,38 @@
...
     @mainTemplate(
         title = intentionToControlP5ViewModel.titleAndHeading,
         showBackLink = true
     ) {
 
-    @heading(
-        heading = intentionToControlP5ViewModel.titleAndHeading,
-        caption = intentionToControlP5ViewModel.lrn.map(x => messages("departure.messages.caption", x))
-    )
+    @formHelper(action = controllers.departureP5.routes.IntentionToControlP5Controller.onSubmit(departureId, messageId), Symbol("autoComplete") -> "off") {
```
Comment (jackheslop96) [inline app/views/departureP5/IntentionToControlP5View.scala.html:40]: Again very minor but if we're wanting to sort out the indentation here it looks like the mainTemplate block can be un-indented 😄

File: test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala (hunk 1 above)
Comment (justega247) [reply test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala:141]: You're right, it is a leftover from copy and pasting.

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**